lib/rails_admin/extension.rb

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
`Layout/LeadingBlankLines: Unnecessary blank line at the beginning of the source.`